js/apps/system/_admin/aardvark/APP/react/yarn.lock

```diff
@@ -8019,9 +8019,9 @@ json5@^2.1.2, json5@^2.2.0:
   dependencies:
```

Comment:
since we are using `yarn` as a build tool, let's remove `package-lock.json` (which as just added in this PR)